regression/apps/scripts/network.sh

```diff
@@ -18,7 +18,6 @@ echo "Start network test......"
 ./socketpair
 ./sockoption
 ./listen_backlog
-./send_buf_full
```

Comment:
Could you please add detailed instructions on how to reproduce the test failure on your Linux machine? This test works pretty fine on my end.
```
> gcc send_buf_full.c -o /tmp/x
> /tmp/x
Start receiving...
Sent bytes: 2577921
Received bytes: 2577921
Test passed: Equal sent bytes and received bytes
```

It's very strange that in your test failure (as provided in the PR description) there is a large gap between the bytes sent and the bytes received. I cannot understand how this can happen at all.

Reply:
@lrh2000, I open an issue #646 for how to reproduce the result.